src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/AbstractRequest.java

```diff
@@ -438,13 +438,107 @@ public ResourceLocation getDisplayIcon()
     }
 
     @NotNull
+    @Override
+    public List<ItemStack> getDisplayStacks()
+    {
+        if (!(getRequest() instanceof IDeliverable))
+        {
+            return Lists.newArrayList();
+        }
+
+        final IDeliverable deliverable = (IDeliverable) getRequest();
+
+        if (itemExamples == null)
+        {
+            itemExamples = ImmutableList.copyOf(StreamSupport.stream(Spliterators.spliteratorUnknownSize(Item.REGISTRY.iterator(), Spliterator.ORDERED), false)
+                    .flatMap(item -> {
+                        NonNullList<ItemStack> stacks = NonNullList.create();
+                        try
+                        {
+                            item.getSubItems(item, null, stacks);
+                        }
+                        catch (Exception ex)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1166)